HoneyBadgers.WebApp/HoneyBadgers.WebApp.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Controllers\HomeController1.cs" />
```

Comment:
Co to robi? :smile:

Reply:
no idea. To było związane z tymi 2 controlerami które się utworzyły przez przypadek. usunąłem to